pkg/kubelet/pleg/generic.go

```diff
@@ -388,18 +435,17 @@ func (g *GenericPLEG) getPodIPs(pid types.UID, status *kubecontainer.PodStatus)
 	return oldStatus.IPs
 }
 
-func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) error {
+func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) (error, bool) {
```

Comment:
can you either document in a comment or in a named return value what this boolean is?

Reply:
Added a method doc. Thanks.